CMakeLists.txt

```diff
@@ -614,10 +614,15 @@ add_subdirectory(build_tools/embed_data/)
 add_subdirectory(tests EXCLUDE_FROM_ALL)
 
 if(IREE_BUILD_BENCHMARKS)
-  find_program(IREE_IMPORT_TFLITE_PATH iree-import-tflite)
-  if(IREE_IMPORT_TFLITE_PATH)
-    message(STATUS "Found ${IREE_IMPORT_TFLITE_PATH} to generate benchmark artifacts")
+  if (IREE_EXPLICIT_IMPORT_TFLITE_PATH)
+    set(IREE_IMPORT_TFLITE_PATH "${IREE_EXPLICIT_IMPORT_TFLITE_PATH}")
   else()
+    find_program(IREE_IMPORT_TFLITE_PATH iree-import-tflite)
```

Comment:
Does `CMAKE_PROGRAM_PATH` work for your intended usage? It would be nice to just use CMake's built in functionality instead of adding even more branches and options custom to our project.

https://cmake.org/cmake/help/latest/command/find_program.html
See the section with this introduction:
> If NO_DEFAULT_PATH is specified, then no additional paths are added to the search. If NO_DEFAULT_PATH is not specified, the search process is as follows:

Reply:
yeah, best bet would be to use that or just put the tool on your path (cmake will look first at things like CMAKE_PROGRAM_PATH but then goes to the PATH env var, so you can do `PATH=tool/dir/:$PATH cmake configure ...`)

Reply:
thanks for the replies, closing this.